EntityLib/Exception.h

```diff
@@ -20,39 +20,41 @@ namespace std
 
 #undef COMPILER_VERSION
 
-char const* formatPath(std::filesystem::path const& _base, std::filesystem::path const& _rel);
+namespace Ent{
 
```

Comment:
This is not the right formatting.
Could you please run clang-format on the changed files ? 
(If you use **visual studio**, **clang power tools** allow to do this easily)

Reply:
Or just replace by 
```cpp
namespace Ent
{
```
should be ok.